fuselage.py

```diff
@@ -227,12 +228,12 @@ def setup(self, Nmissions, **kwargs):
         constraints = []
         with SignomialsEnabled():
             constraints.extend([
-
                 # Passenger constraints
                 Wlugg >= flugg2 * npass * 2 * Wchecked + flugg1 * npass * Wchecked + Wcarryon,
-                Wpass == npass * Wavgpass,
+                Wpass >= npass * Wavgpass,
                 Wpay >= Wpass + Wlugg + Wcargo,
                 Wpay >= npass * Wavgpasstot,
+                Wpaymax >= Wpay,
                 nseat >= npass,
```

Comment:
TCS here too

Reply:
Hmmm how about no, because it's just going to throw unnecessary warnings every time we run multimission.

Reply:
sounds good